sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeHealthcareEntitiesTests.cs

```diff
@@ -54,6 +54,8 @@ public AnalyzeHealthcareEntitiesTests(bool isAsync, TextAnalyticsClientOptions.S
         };
 
         [RecordedTest]
+        // Temporarily setting max version to V2022-05-01 since V2022-10-01-preview does not support AAD yet.
```

Comment:
I will create an issue to keep track of removing this ServiceVersion.Max attribute once support for AAD is enabled for version 2022-10-01-preview.